docs/introduction/start-building-your-application.md

```diff
@@ -48,6 +48,35 @@ When you install new project, locales are set like this
 
 In case you want to change domain locale read [locale settings](./how-to-set-up-domains-and-locales.md#3-locale-settings) or in case you want to change default administration locale read [locale in administration](./how-to-set-up-domains-and-locales.md#36-locale-in-administration).
 
+## Set up domain type
+
+Each domain can be configured with a type. The type can be either `B2C` (Business-to-Consumer) or `B2B` (Business-to-Business). This option is set using the `type` parameter in the domain configuration.
+
+By default, if the `type` parameter is not present, the domain is considered as `B2C`.
+
+Here is an example of how to set up the domain type:
+
+```yaml
+domains:
+    - id: 1
+      name: shopsys
+      locale: en
+      url: http://127.0.0.1:8000
+      type: B2C
```

Comment:
Does the letter case matter? in code is it lowercased, here uppercased

Reply:
It does matter. I change configuration ? Should I mention case sensitivity ?

Reply:
*https://github.com/shopsys/shopsys/pull/3286/commits/a25913ef278b9cd645b3d01d0f5bbcf44bd54bbf

Reply:
> Should I mention case sensitivity ?

I don't think it's necessary